src/PointwiseFunctions/GeneralRelativity/ComputeGhQuantities.hpp

```diff
@@ -129,4 +129,62 @@ tnsr::ii<DataType, SpatialDim, Frame> extrinsic_curvature(
 template <size_t SpatialDim, typename Frame, typename DataType>
 tnsr::ijj<DataType, SpatialDim, Frame> deriv_spatial_metric(
     const tnsr::iaa<DataType, SpatialDim, Frame>& phi) noexcept;
+
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Computes spatial derivatives of lapse (N) from the generalized
+ *        harmonic variables and spacetime unit normal 1-form.
+ *
+ * \details If the generalized harmonic conjugate momentum and spatial
+ * derivative variables are \f$\pi_{ab} = -n^c \partial_c \psi_{ab} \f$ and
+ * \f$\phi_{iab} = \partial_i \psi_{ab} \f$, the spatial derivatives of N
+ * can be obtained from:
+ * \f{align*}
+ *  n^a n^b \phi_{iab} = -1/(2N) [\partial_i (-N^2 + N_jN^j)-
+ *                               2 N^j \partial_i N_j
+ *                               + N^j N^k \partial_i g_{jk}]
+ *                     = -(2/N) \partial_i N
+ * \f},
```

Comment:
comma should be on line with equation, `since` on its own line, and the final equation on `\f[` `\f]`.